hera/nfs/workflow.py

```diff
@@ -0,0 +1,23 @@
+from hera.workflows import Workflow, Parameter, DAG, Container, Task, Resource
+from ci_workflow import ci_workflow
+
+import shared
+
+with Workflow(
+    generate_name="hera-ci-workflow-",
+    namespace="argo",
```

Comment:
`namespace` does not need to be added here if it's added through `global_config.namespace`.

Reply:
Done